pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -21,11 +22,12 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against nodes.
-func NewStorage(s storage.Interface) *REST {
+func NewStorage(optsGetter restoptions.Getter) *REST {
```

Comment:
we want to look up options for a specific resource, so we can do things like shard etcd, have different watch cache enablement or size settings, etc.